tracks/python/exercises/darts/mentoring.md

```diff
@@ -0,0 +1,93 @@
+# Mentoring
+
+## Problem and Challenges
+
+The problem asks you to write a function which returns earned points in a single toss of a Darts game.
+The dart's location is given as a Cartesian point (an `x` and `y` value).
+Points are based on the distance from the center.
+
+## Reasonable Solution
+
+```python
+import math
+
+
+def score(x, y):
+    distance = math.sqrt(x * x + y * y)
+    if distance <= 1:
+        return 10
+    if distance <= 5:
+        return 5
+    if distance <= 10:
+        return 1
+    return 0
+```
+
+## Talking Points
+
+### Avoiding the use of `else`
+
+It can be tempting to use `elif` for the conditionals after the first one.
+However, doing so would be an "unnecessary else after return", as flagged by Pylint ([R1705][R1705]).
+
+### Minimizing the number of comparisons
+
+The distance checks can comute the upper and lower bound for each point band.
```

Comment:
While comute is misspelled, but also it may be misleading.  Is it supposed to be commute or compute?  Both meanings might be applied here, depending on what the exchange would result in.

Reply:
That was supposed to be `compute`.